### PR TITLE
content library testing

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY
Test infra for content library

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE
- Feature Pull Request

### CODE REVIEW

File: tests/integration/targets/content_library/tasks/main.yaml [hunk 1]
```diff
@@ -0,0 +1,102 @@
+---
+- set_fact:
+    datastore_name: "rw_datastore"
+    local_library_name: "local_library_001"
+
+- name: Get Datastore id required for local library creation
+  vmware.vmware_rest.vcenter_datastore_info:
+    filter_names:
+      - "{{ datastore_name }}"
+  register: r
```
Comment (goneri) [inline tests/integration/targets/content_library/tasks/main.yaml:10]: I use `result` in all the other tests. Do you mind if we use the same variable name in this playbook too?

Comment (Akasurde) [reply tests/integration/targets/content_library/tasks/main.yaml:10]: OK.

File: tests/integration/targets/content_library/tasks/main.yaml [hunk 2]
```diff
@@ -0,0 +1,102 @@
+---
+- set_fact:
+    datastore_name: "rw_datastore"
+    local_library_name: "local_library_001"
+
+- name: Get Datastore id required for local library creation
```
Comment (goneri) [inline tests/integration/targets/content_library/tasks/main.yaml:6]: Can you add a `_` to the task `name`, this way the task won't be injected in `vcenter_datastore_info`'s `DOCUMENTATION` block.

**Review by goneri — CHANGES_REQUESTED** [review]

**Review by goneri — APPROVED** [review]

**Comment by goneri** [comment]
Thank you @Akasurde :-)

**Pull request merged**